src/main/java/by/custom/utilcalculator/exception/UtilsborException.java

```diff
@@ -0,0 +1,20 @@
+package by.custom.utilcalculator.exception;
+
+import org.telegram.telegrambots.meta.api.objects.Message;
+
+public abstract class UtilsborException extends Exception {
+    public static final String USER_FILE_NOT_FOUND = "user_file_not_found";
```

Comment:
You could move it to enum `UtilsborErrorCode`

Reply:
Done.